policies/entity_stability.rego

```diff
@@ -0,0 +1,48 @@
+package after_resolution
+import rego.v1
+
+# checks for stable entity groups that have no identifying attributes
+deny contains entity_stability_violation(description, group.id, "") if {
+    group := input.groups[_]
+    # ignore attribute_groups
+    group.type == "entity"
+    group.stability == "stable"
+
+    exceptions = {}
+    not exceptions[group.id]
+
+    ids := [attr |
+        some attr in group.attributes
+        attr.role == "identifying"
+    ]
+    count(ids) < 1
+
+    description := sprintf("Stable entity '%s' has no identifying attributes", [group.name])
```

Comment:
do you think it should be a policy or a hard schema requirement? I.e. should become part of weaver in the long run?